deployment-transformer/deployment-transformer-impl/pom.xml

```diff
@@ -69,8 +69,8 @@
             <artifactId>deployment-transformer-api</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.eclipse.transformer</groupId>
-            <artifactId>org.eclipse.transformer.payara</artifactId>
+            <groupId>fish.payara.transformer</groupId>
+            <artifactId>fish.payara.transformer.payara</artifactId>
```

Comment:
We don't need to change the artifact ID as well do we?

Reply:
I talked with @jGauravGupta and we decided to also change the artifacts to make compatible the new group id with the artifact names. If you suggest something different we can review and make the changes